src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      {person.isMarried
+        && (person.sex === 'm'
+          ? `${person.partnerName} is my wife`
+          : `${person.partnerName} is my husband`)}
+
+      {!person.isMarried && 'I am not married'}
```

Comment:
suggest to make more readable and shorter this part.
before return jsx in component we can decide which text will be rendered.
also suggest to destructure person;
const { name, age, sex, isMarried, partnerName } = person;
```
const personPartner = sex === 'm' ? 'wife' : 'husband';
const personPartnerText = isMarried ? `${person.partnerName} is my ${personPartner}` : 'I am not married';
```
```suggestion
      {personPartnerText}
```